src/test/test-cpu-set-util.c

```diff
@@ -277,4 +277,23 @@ TEST(print_cpu_alloc_size) {
         log_info("CPU_ALLOC_SIZE(8191) = %zu", CPU_ALLOC_SIZE(8191));
```

Comment:
Regarding adding tests for the entire feature, where would they go? I looked around, but didn't find any comparable tests - i.e for top-level options like this. But maybe I missed something.